config/dev/dgutman-dsa-server-preprod-ec2.yaml

```diff
@@ -0,0 +1,27 @@
+# This stack is based on `config/prod/dgutman-htan-new2-ec2.yaml`
+# in https://github.com/Sage-Bionetworks/sandbox-provisioner/
+template_path: "remote/managed-ec2-linux-v2.j2"
+stack_name: "dgutman-dsa-server-preprod-ec2"
+
+sceptre_user_data:
+  Distro: "ubuntu"
+  OpenPorts: ["80", "443", "22"]
+parameters:
+  Department: "CompOnc"
+  Project: "htan"
+  OwnerEmail: "bruno.grande@sagebase.org"
+  InstanceType: "i3en.2xlarge"
+  AMIId: "ami-0b7906ab614596e7e"  # https://github.com/Sage-Bionetworks-IT/packer-base-ubuntu-bionic/releases/tag/v1.0.9
+  VolumeSize: "1000"
+  KeyName: "EmoryHealthcare-dgutman"  # Imported via AWS Management Console
+  VpcName: "htan-vpc"
+  VpcSubnet: "PublicSubnet"
```

Comment:
@ahayden should review your intention here.

Reply:
It looks to me like this server is used for production work (despite it being called dev), in which case it should not be in a public subnet. This is a very fragile deployment and is a bad practice to use a single host as web server and jumpbox.

Reply:
If HTAN can tolerate the fragility, does this block the PR?

Reply:
If it's a prototype, I don't have a problem with it... HTAN does not get to define Sage's security practices... The project is either a prototype, in which is is not used for production. Or, is something that we will support as production work, in which case it needs to be designed according to IT Ops standards.